pkg/geodata/geodata.go

```diff
@@ -150,6 +153,77 @@ AND nomis_category.year = %d
 	return app.collectCells(ctx, sql)
 }
 
+// radiusQuery returns the csv table for areas within radius meters from location
+//
+func (app *Geodata) radiusQuery(ctx context.Context, location string, radius int, geotype string, cats []string) (string, error) {
+	var lon, lat float64
+	fields, err := fmt.Sscanf(location, "%f,%f", &lon, &lat)
+	if err != nil {
+		return "", fmt.Errorf("scanning location %q: %w", location, err)
+	}
+	if fields != 2 {
+		return "", fmt.Errorf("location missing a number: %w", ErrMissingParams)
+	}
+
+	if radius < 1 {
+		return "", fmt.Errorf("radius must be >0: %q: %w", radius, err)
+	}
+
+	if geotype == "" {
```

Comment:
maybe OOS for this, but I imagine there might be a usecase in which multiple geo_types are needed?